spring-cloud-stream-binder-kafka-streams/src/main/java/org/springframework/cloud/stream/binder/kafka/streams/properties/KafkaStreamsStateStoreProperties.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.binder.kafka.streams.properties;
+
+
+/**
+ * @author Lei Chen
+ */
+public class KafkaStreamsStateStoreProperties {
```

Comment:
Why does this class need to be `public` and part of the API? It is only used internally. Maybe you can move it to the top level package and make the access level `package-private`?

Reply:
Doing that will prevent KafkaStreamsStateStore in a different package from using it. I think it's ok to leave it there as public, which allows us to support property driven state store creation in future.

Reply:
ok